common/ansi/src/main/java/io/helidon/build/common/ansi/AnsiConsoleInstaller.java

```diff
@@ -159,6 +159,11 @@ private Holder() {
                 Ansi.setEnabled(false);
             } else {
                 ConsoleType desiredType = desiredConsoleType();
+                String arch = System.getProperty("os.arch");
```

Comment:
Can you add this comment from the pico code `// workaround for https://github.com/fusesource/jansi/issues/162`